src/providers/fault-summary/fault-summary.ts

```diff
@@ -326,4 +391,60 @@ export class FaultSummaryProvider {
     }
     return result;
   }
+
+  private getVehicleCheckDrivingFaultsCatC(vehicleChecks: CatCUniqueTypes.VehicleChecks, category: CategoryCode)
```

Comment:
think this will have an issue where there are 5 show me tell me question driving faults. 
This would result in it returning driving faults - rather than it seeing that there is a serious fault and exiting.

Reply:
I think vehicleCheckFaults will have serious faults returned so this could be checked and returned.

Reply:
@rossfauldsbjss i've replicated the logic that exists in BE, and the Cat BE/C have the same number of vehicle checks, so is the logic not the same?

Reply:
It looks like a logic flaw to me - in that it exits if there is a serious or dangerous fault, but in the case of 5 df this is considered a serious.  I think the flaw is in both, however depending on what uses this method it may not manifest as an issue.  So not sure.  I think it looks to me like it would be used to bypass editing of fault comments on the office page.  Could you check what uses it please?